evap/staff/tests/test_views.py

```diff
@@ -2432,19 +2436,27 @@ def setUpTestData(cls):
 
     def test_emailtemplate(self):
         """
-            Tests the emailtemplate view with one valid and one invalid input datasets.
+            Tests the emailtemplate view with one valid and two invalid input datasets.
         """
         page = self.app.get(self.url, user=self.manager, status=200)
         form = page.forms["template-form"]
         form["subject"] = "subject: mflkd862xmnbo5"
-        form["body"] = "body: mflkd862xmnbo5"
+        form["plain_body"] = "plain_body: mflkd862xmnbo5"
+        form["html_body"] = "html_body: <p>mflkd862xmnbo5</p>"
         form.submit()
 
-        self.assertEqual(EmailTemplate.objects.get(pk=1).body, "body: mflkd862xmnbo5")
+        self.assertEqual(EmailTemplate.objects.get(pk=1).plain_body, "plain_body: mflkd862xmnbo5")
+        self.assertEqual(EmailTemplate.objects.get(pk=1).html_body, "html_body: <p>mflkd862xmnbo5</p>")
 
-        form["body"] = " invalid tag: {{}}"
+        form["plain_body"] = " invalid tag: {{}}"
         form.submit()
-        self.assertEqual(EmailTemplate.objects.get(pk=1).body, "body: mflkd862xmnbo5")
+        self.assertEqual(EmailTemplate.objects.get(pk=1).plain_body, "plain_body: mflkd862xmnbo5")
+        self.assertEqual(EmailTemplate.objects.get(pk=1).html_body, "html_body: <p>mflkd862xmnbo5</p>")
+
+        form["html_body"] = ""
+        form.submit()
+        self.assertEqual(EmailTemplate.objects.get(pk=1).plain_body, "plain_body: mflkd862xmnbo5")
+        self.assertEqual(EmailTemplate.objects.get(pk=1).html_body, "html_body: <p>mflkd862xmnbo5</p>")
```

Comment:
Sooo we can't delete the html_body? Is this what we want? I expected an invalid tag test here, just as it was with the plain body.